birdbox/microsite/migrations/0068_alter_generalpurposepage_content_and_more.py

```diff
@@ -144,7 +144,7 @@ class Migration(migrations.Migration):
                                     "alignment",
                                     wagtail.blocks.ChoiceBlock(
                                         blank=True,
-                                        choices=[("", "Center"), ("mzp-c-section-heading-left", "Left"), ("mzp-c-section-heading-right", "Right")],
+                                        choices=[("", "Default"), ("t-align-center", "Center"),],
```

Comment:
Editing migrations for streamblocks is ok, but it's often simpler to make a new one with `just makemigrations`